src/Makefile_obj.in

```diff
@@ -67,6 +67,7 @@ CFG_CXXFLAGS_PARSER = @CFG_CXXFLAGS_PARSER@
 CFG_CXXFLAGS_WEXTRA = @CFG_CXXFLAGS_WEXTRA@
 CFG_LDFLAGS_SRC = @CFG_LDFLAGS_SRC@
 CFG_LIBS = @CFG_LIBS@
+CFG_WITH_THREADED = @CFG_WITH_THREADED@
```

Comment:
This doesn't exist in configure.   We always require C++11 so don't think this/the below ifeq needs conditional.